core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5588,6 +5588,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey S3_REST_AUTHENTICATION_ENABLED =
+      new Builder(Name.S3_REST_AUTHENTICATION_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("Whether to enable check s3 rest request header.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey S3_SECRET_MANAGER_CLASSNAME =
+      new Builder(Name.S3_SECRET_MANAGER_CLASSNAME)
+          .setDescription("The class to provide customized s3 secret manager implementation which"
+                 + "implement the interface 'alluxio.proxy.s3.signature.S3SecretManager'. "
+                 + "When alluxio.s3.rest.authentication.enabled is set to true, it must be set.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.ALL)
+          .build();
```

Comment:
I appreciate the idea of having pluggable S3 secret management, however I don't think the requirements are well-defined enough to have an interface at the moment. As such I'd rather not pre-emptively add this feature; maybe add some comments mentioning that authorization currently does not have a secrets store.

If you are using this interface as part of your local environment on a separate branch, please move this feature onto that branch instead. @HelloHorizon @yuzhu wdyt?